vmdb/app/models/ems_kubernetes.rb

```diff
@@ -11,19 +11,40 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
-  def self.raw_connect(hostname, port, api_version)
+  def self.raw_connect(uri, api_version)
```

Comment:
Hmmm...I see what you are trying to do... You are building the URI, just not here.  I'm on the fence.  I use raw_connect for diagnosis or raw access sometimes, and having the hostname and port separately seems preferable...and so I don't have to remember the "extra" details, like "/api"